manifest.php

```diff
@@ -34,7 +34,7 @@
     'label' => 'Test-taker core extension',
     'description' => 'TAO TestTaker extension',
     'license' => 'GPL-2.0',
-    'version' => '7.2.2',
+    'version' => '7.2.3',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => [
         'tao' => '>=36.1.0',
```

Comment:
If we require specific minimum version of some extension (in this PR we need never version `41.1.1` of tao, with new method in UserService), it should be specified here.